assets/js/modules/analytics/components/common/AdSenseLinkCta.js

```diff
@@ -24,9 +24,9 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import CTA from '../../../components/notifications/cta';
+import CTA from '../../../../components/notifications/cta';
 
-export default function AdSenseLinkCTA() {
+export default function AdSenseLinkCta() {
```

Comment:
Per our naming conventions (all abbreviations fully capitalized), this should continue to be called `AdSenseLinkCTA`.

Reply:
Ah ok, the pascal case setting in eslintrc.json wasn't allowing those uppercased acronyms, but I'll swap it out for a regex that should allow it.